packages/ubuntu_desktop_installer/lib/pages/updates_other_software/updates_other_software_model.dart

```diff
@@ -35,4 +40,18 @@ class UpdateOtherSoftwareModel extends ChangeNotifier {
 
     notifyListeners();
   }
+
+  /// Select the source corresponding to the selected installation mode.
+  Future<void> selectInstallationSource() {
+    String sourceId;
+    switch (installationMode) {
+      case InstallationMode.normal:
+        sourceId = 'ubuntu-desktop';
+        break;
+      case InstallationMode.minimal:
+        sourceId = 'ubuntu-desktop-minimal';
+        break;
+    }
```

Comment:
I like to make enum extensions for enum->string conversions - makes the code pretty :slightly_smiling_face:

Reply:
Good suggestion, I did that.